lib/resolveurl/plugins/avideo.py

```diff
@@ -27,5 +27,5 @@ class AVideoResolver(ResolveGeneric):
 
     def get_media_url(self, host, media_id):
         return helpers.get_media_url(self.get_url(host, media_id),
-                                     patterns=[r'''file:\s*"(?:\[\w*\])?(?P<url>[^"]+)",'''],
+                                     patterns=[r'''sources:\s*\[{src:\s*['"](?P<url>[^'"]+)'''],
```

Comment:
Please add the new pattern as an additional element in the patterns list, so that it will work in both cases. There may be still embeds using the other pattern

Reply:
Done. I just wrote this 2 weeks ago, they all switched. Pretty sure of it. but updated anyways